server/gameconnection.py

```diff
@@ -223,7 +223,10 @@ async def handle_game_option(self, key, value):
             return
 
         if key == "Victory":
-            self.game.gameOptions["Victory"] = Victory.from_gpgnet_string(value)
+            try:
+                self.game.gameOptions["Victory"] = Victory[value.upper()]
+            except KeyError:
+                self.game.gameOptions["Victory"] = None
```

Comment:
`Victory.__members__.get("DEMORALIZATION", None)`

Reply:
Hm, I like this, but I'm not sure whether we want to rely on the `__members__` implementation. As far as I'm aware the library retains the right to change them as they see fit, as if they were private in other programming languages.

Reply:
I think it's fine. I read through the Enum source code a while back and I think it's mostly a dunder to signify that it is a special member and not an enum variant. There are some other ones too like `__order__` and they are meant to be used by client code. In python usually having underscores in the front and back means `magic` not `private` (like `__init__`, `__eq__`, etc).

Reply:
Alright!